drivers/usb/device/usb_dc_native_posix_adapt.c

```diff
@@ -223,10 +235,8 @@ static void handle_usbip_unlink(int connfd, struct usbip_header *hdr)
 		return;
 	}
 
-	/* Read also padding */
-	read = recv(connfd, &setup_padding, sizeof(setup_padding), 0);
-	if (read != sizeof(setup_padding)) {
-		LOG_ERR("recv() failed: %s", strerror(errno));
+	if (!usbip_skip_setup()) {
```

Comment:
Could You please provide explanation in commit message why skip is necessary here?

Reply:
I added comment, /* Read USB setup, not handled */
I do not know what really should I add more, the function is self-describing.